cassandra/metadata.py

```diff
@@ -140,7 +160,7 @@ def rebuild_schema(self, ks_results, type_results, cf_results, col_results, trig
                 self._keyspace_added(keyspace_meta.name)
 
         # remove not-just-added keyspaces
-        removed_keyspaces = [ksname for ksname in self.keyspaces.keys()
+        removed_keyspaces = [name for name in self.keyspaces.keys()
                              if ksname not in current_keyspaces]
```

Comment:
Remembered -- I think I was trying to silence a flake8 warning about a name collision